### PR TITLE
test: increased code coverage for badfish.py

### PR DESCRIPTION
**Pull request by k3nd1k**

### CODE REVIEW

File: tests/test_ls_memory.py [hunk 1]
```diff
@@ -16,13 +21,59 @@ class TestLsMemory(TestBase):
     @patch("aiohttp.ClientSession.get")
     def test_ls_memory(self, mock_get):
         responses_add = [
-            MEMORY_SUMMARY_RESP,
-            MEMORY_MEMBERS_RESP,
-            MEMORY_A5_RESP,
-            MEMORY_B2_RESP,
+            MEMORY_SUMMARY_RESP,  # beginning in get_memory_summary()
```
Comment (grafuls) [inline tests/test_ls_memory.py:24]: remove comments

File: tests/test_ls_memory.py [hunk 2]
```diff
@@ -16,13 +21,59 @@ class TestLsMemory(TestBase):
...
+        responses = INIT_RESP + responses_add
+        self.set_mock_response(mock_get, 200, responses)
+        self.args = [self.option_arg]
+        _, err = self.badfish_call()
+        assert err == RESPONSE_LS_MEMORY_SUMMARY_PROC_DATA_ERROR
+
+    @patch("aiohttp.ClientSession.get")
+    def test_ls_memory_details_not_found(self, mock_get):
+        responses_add = [
+            MEMORY_SUMMARY_RESP,  # beginning in get_memory_summary()
```
Comment (grafuls) [inline tests/test_ls_memory.py:60]: remove comments

File: tests/test_ls_memory.py [hunk 3]
```diff
@@ -16,13 +21,59 @@ class TestLsMemory(TestBase):
...
+        responses = INIT_RESP + responses_add
+        self.set_mock_response(mock_get, [200, 200, 200, 200, 404], responses)
+        self.args = [self.option_arg]
+        _, err = self.badfish_call()
+        assert err == RESPONSE_LS_MEMORY_DETAILS_NOT_FOUND
+
+    @patch("aiohttp.ClientSession.get")
+    def test_ls_memory_details_value_error(self, mock_get):
+        responses_add = [
+            MEMORY_SUMMARY_RESP,  # beginning in get_memory_summary()
```
Comment (grafuls) [inline tests/test_ls_memory.py:72]: remove comment

File: tests/test_reset_idrac.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
     RESET_TYPE_RESP,
     BOOT_SEQ_RESPONSE_DIRECTOR,
     RESPONSE_RESET,
-    INIT_RESP,
+    INIT_RESP, RESPONSE_RESET_FAIL,
```
Comment (grafuls) [inline tests/test_reset_idrac.py:6]: separate lines

File: tests/test_ls_memory.py (hunk 1 above)
Comment (k3nd1k) [reply tests/test_ls_memory.py]: Fixed it with the latest commit.

File: tests/test_ls_memory.py (hunk 2 above)
Comment (k3nd1k) [reply tests/test_ls_memory.py]: Fixed it with the latest commit.

File: tests/test_ls_memory.py (hunk 3 above)
Comment (k3nd1k) [reply tests/test_ls_memory.py]: Fixed it with the latest commit.

File: tests/test_reset_idrac.py (hunk 4 above)
Comment (k3nd1k) [reply tests/test_reset_idrac.py]: Fixed it with the latest commit.

**Pull request merged**